src/components/ec-radio-btn/types.ts

```diff
@@ -0,0 +1,20 @@
+export enum RadioButtonEvent {
+    UPDATE_MODEL_VALUE = 'update:modelValue',
+  }
+
+export interface RadioButtonEvents {
+    [RadioButtonEvent.UPDATE_MODEL_VALUE]: RadioButtonProps['modelValue']
+  }
+
+export interface RadioButtonProps {
+    modelValue?: string,
+    label?: string,
+    description?: string,
+    isDisabled?: boolean,
+    isTextInline?: boolean
+    value: string,
+    name?: string,
+    errorMessage?: string,
+    hasError?: boolean,
+  }
```

Comment:
```suggestion
export interface RadioButtonProps {
  value: string,
  modelValue?: string,
  label?: string,
  description?: string,
  isDisabled?: boolean,
  isTextInline?: boolean,
  name?: string,
  errorMessage?: string,
  hasError?: boolean
}
```
Please copy interface from the `.vue` file, to have them equal and easy comparable. Don't remember `export` word here and do not have one in the `.vue` file.